spec/lib/orcid/pub_mapper_spec.rb

```diff
@@ -168,6 +168,34 @@
     end
   end
 
+  context 'with abstract greater than 5000 characters' do
+    let(:big_abstract) { SecureRandom.random_number(36**6000).to_s(36) } # generates 6000 character random string
+
+    let(:pub_hash) do
+      base_pub_hash.dup.tap do |pub_hash|
```

Comment:
`.dup.tap` is basically same as `base_pub_hash.merge(abstract: big_abstract)`.  I think the merge format is more readable 🤷

Reply:
makes sense -- i switched it